crates/router/src/services/api.rs

```diff
@@ -1933,6 +1933,19 @@ pub fn build_redirection_form(
                 }
             }
         },
+        RedirectForm::KlarnaCheckout { html_snippet } => {
+            maud::html! {
```

Comment:
please use the existing html variant, let's not create a separate HTML specifically for klarna if possible